### PR TITLE
FEAT: Terms of Service & Privacy Policy Change <ENG-2538>

### PR DESCRIPTION
**Pull request by 0xMuTwo**
# Task:

`/terms-of-service` and `/privacy-policy` routes are hidden behind the location blocker. 
## Description

Alterations to restricted blocker for `/terms-of-service` and `/privacy-policy`.

Fixes (ENG-2538)

## Type of change

- [ ] New feature
- [X] Bug fix
- [ ] Testing code
- [ ] Document update or config files

### CODE REVIEW

**Review by nikkaroraa — APPROVED** [review]
looks good. lets merge it 🚀

File: packages/frontend/src/components/StrikeWarning.tsx [hunk 1]
```diff
@@ -89,18 +90,22 @@ export default function StrikeModalManager() {
...
+  const router = useRouter()
+
+  // Check if the current page is either terms of service or privacy policy
+  const isExcludedPage = ['/terms-of-service', '/privacy-policy'].includes(router.pathname)
 
   // delayed state change - show modal after 2 seconds
   // this is because it takes some time to automatically connect the wallet (incase previously connected)
   useEffect(() => {
-    if (isRestricted) {
+    if (isRestricted && !isExcludedPage) {
```
Comment (nikkaroraa) [inline packages/frontend/src/components/StrikeWarning.tsx:101]: i don't think we need to check `isExcludedPage` here. 

`if (showModal && !isExcludedPage)` this should be enough

**Review by nikkaroraa — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/StrikeWarning.tsx (hunk 1 above)
Comment (0xMuTwo) [reply packages/frontend/src/components/StrikeWarning.tsx:101]: Awesome catch, you're right this is redundant. Fixed in new version.

**Review by nikkaroraa — APPROVED** [review]
looks good - merging

**Pull request merged**